### PR TITLE
VL-94_Table-bugs-and-overflow-props_Dmytro-Holdobin

### PR DESCRIPTION
**Pull request by Explicit12**
> https://ilevel.atlassian.net/browse/VL-94
> https://ilevel.atlassian.net/browse/VL-96

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -15,10 +15,12 @@ import { createVueless } from "../src/index.js";
 import { createRouter, createWebHistory } from "vue-router";
 
 /* Setup storybook */
-setup((app) => {
+setup((app, context) => {
   const vueless = createVueless();
   const router = createRouter({ history: createWebHistory(), routes: [] });
 
+  app.config.idPrefix = context.name;
```
Comment (itsJohnnyGrid) [inline .storybook/preview.js:22]: For what reason this line is needed?

File: src/ui.data-table/UTableRow.vue [hunk 2]
```diff
@@ -31,21 +31,27 @@
...
+          ref="toggle-wrapper"
+          v-bind="bodyCellNestedExpandIconWrapperAttrs"
+          :style="{ width: getIconWidth() }"
+        >
+          <UIcon
+            v-if="isShownToggleIcon"
+            size="xs"
+            internal
+            interactive
+            :data-row-toggle-icon="`row-toggle-icon-${row.id}`"
```
Comment (itsJohnnyGrid) [inline src/ui.data-table/UTableRow.vue:44]: maybe do it simpler:

`:data-toggle-icon-row="row.id"`

File: .storybook/preview.js (hunk 1 above)
Comment (Explicit12) [reply .storybook/preview.js:22]: Each story is vue app instance. We have to define id prefix for useId when we have more than one app in the page.

File: src/ui.data-table/UTableRow.vue (hunk 2 above)
Comment (Explicit12) [reply src/ui.data-table/UTableRow.vue]: It's still toggle button of the row, not the whole row, so I left the name

**Review by itsJohnnyGrid — APPROVED** [review]

**Pull request merged**